app/scripts/views/confirm.js

```diff
@@ -45,6 +45,7 @@ define(function (require, exports, module) {
       var email = this.getAccount().get('email');
       var isSignIn = this.isSignIn();
       var isSignUp = this.isSignUp();
+      var openWebmailButtonVisible = this.isOpenWebmailButtonVisible(email);
```

Comment:
we can remove the `isSignUp` and move it to return , others are fine , they are used multiple times.